Functions.Templates/Templates/DurableFunctionsActivity-CSharp/build.config/template.json

```diff
@@ -0,0 +1,58 @@
+﻿{
+  "author": "Microsoft",
+  "classifications": [
+    "Azure Function"
+  ],
+  "name": "DurableFunctionsActivityCSharp",
+  "identity": "Azure.Function.CSharp.DurableFunctionsActivityTrigger",
+  "groupIdentity": "Azure.Function.DurableFunctionsActivityTrigger",
+  "shortName": "EventHub",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "sourceName": "DurableFunctionsActivityCSharp",
+  "symbols": {
+    "namespace": {
+      "description": "namespace for the generated code",
+      "replaces": "Company.Function",
+      "type": "parameter"
+    }
+  },
+  "primaryOutputs": [
+    {
+      "path": "DurableFunctionsActivityTrigger.cs"
+    }
+  ],
+  "defaultName": "DurableFunctionsActivityCSharp",
+  "postActions": [
+    {
+      "Description": "Adding Reference to Microsoft.Azure.WebJobs.Extensions.DurableTask Nuget package",
```

Comment:
>Nuget [](start = 89, length = 5)

nit: Rather than "Nuget" we should use the official form "NuGet" here and below.